data/blog/electron-3-0.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Electron 3.0.0
+author: codebytere
+date: '2018-09-20'
+---
+
+The Electron team is excited to annouce that the first stable release of Electron `v3.0.0` is now
+available from [electronjs.org](https://electronjs.org/)! It's jam-packed with upgrades, fixes, and new features, and we can't wait to see what you build with them. Below are more details about the specifics of this latest release, and we welcome your feedback as you explore.
```

Comment:
Might wanna put a one-liner at the top here like `npm i electron@latest --save-dev`